v4/source/system.yaml

```diff
@@ -87,6 +117,21 @@
           $ref: '#/responses/BadRequest'
         '403':
           $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            // GetConfig
+            config, err := Client.GetConfig()
+
+            config.TeamSettings.SiteName = "MyFancyName"
+
+            // UpdateConfig
+            updatedConfig, err := Client.UpdateConfig()
```

Comment:
This should be `Client.UpdateConfig(config)`, right?

Reply:
yes true! done.